client/settings/view.js

```diff
@@ -6,15 +6,15 @@ import React, { PropTypes } from 'react';
 /**
  * Internal dependencies
  */
-import WCCSettingsForm from './wcc-settings-form';
```

Comment:
Let's also rename `WCCSettingsForm` in `/client/settings/components/settings-form/index.js`.